### PR TITLE
Upgrade SvelteKit to `next.208`

### LINKED ISSUE #1412: Upgrade SvelteKit to `next.208` - Breaking changes
See https://twitter.com/geoffrich_/status/1476658347948457988

### PR DESCRIPTION
**Pull request by mikenikles**
Close #1412.

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1413"></a>

### CODE REVIEW

File: src/lib/components/docs/feedback-widget.svelte [hunk 1]
```diff
@@ -27,7 +27,7 @@
         type,
         emotion: selectedEmotion,
         note,
-        url: `https://${$page.host + $page.path}`,
+        url: $page.url.toString(),
```
Comment (Siddhant-K-code) [inline src/lib/components/docs/feedback-widget.svelte:30]: Why are we converting pageUrl to string?

this should work:

```svelte
url: `https://${$page.url}`
```

File: src/lib/components/open-graph.svelte [hunk 2]
```diff
@@ -13,7 +13,7 @@
     norobots = false,
   } = data || {};
 
-  const url = `https://${$page.host}${$page.path}`;
+  const url = $page.url.toString();
```
Comment (Siddhant-K-code) [inline src/lib/components/open-graph.svelte:16]: Same for this ^

[^Ref](https://github.com/gitpod-io/website/pull/1413/files#r778706778)

File: src/lib/components/docs/feedback-widget.svelte (hunk 1 above)
Comment (Siddhant-K-code) [reply src/lib/components/docs/feedback-widget.svelte:30]: Why are we converting pageUrl to string?

this should work:

```svelte

url: `https://${$page.host + $page.url.pathname}`,

```

**Review by Siddhant-K-code — COMMENTED** [review]
Why Are we converting urlPath to a string?

File: src/lib/components/open-graph.svelte (hunk 2 above)
Comment (mikenikles) [reply src/lib/components/open-graph.svelte:16]: `$page.host` is no longer available as of [1.0.0-next.208](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md#100-next208).

**Comment by mikenikles** [comment]
@Cahllagerfeld Could you review this please and merge if you're ok with it?

**Review by Siddhant-K-code — APPROVED** [review]
Manually Tested the [Preview](https://deploy-preview-1413--gitpod-kumquat.netlify.app/) ✅ 

LGTM 🎉

**Review by Cahllagerfeld — APPROVED** [review]
Nice work! 🚀

**Review by nisarhassan12 — APPROVED** [review]
Looks good to me as well 👍🏿

**Pull request merged**